benchmarks/dynamo/common.py

```diff
@@ -689,8 +726,9 @@ def maybe_mark_profile(*args, **kwargs):
             timings,
         )
 
-    first_headers = ["dev", "name", "batch_size"]
-    first_fields = [current_device, current_name, current_batch_size]
+    current_group = ALL_MODEL_GROUPS.model_group(current_name)
+    first_headers = ["dev", "name", "group", "batch_size"]
```

Comment:
In addition to the new column, is this possible to filter the result dataframe and store Blueberies models in separate CSV files such as `inductor_dynamic_blueberries_amp_inference_cuda_performance.csv` and `inductor_dynamic_blueberries_amp_inference_cuda_accuracy.csv` with `blueberries` in the file name.  Duplicated records are fine, so llama, for example, could appear in both TorchBench CSV file and Blueberries CSV file